front/app/components/StatusBadge/index.tsx

```diff
@@ -30,14 +30,14 @@ interface Props {
 }
 
 const StatusBadge = memo<Props>(({ statusId, id, className }) => {
-  const ideaStatus = useIdeaStatus({ statusId });
+  const { data: ideaStatus } = useIdeaStatus(statusId);
 
   if (!isNilOrError(ideaStatus)) {
```

Comment:
Is `nilOrError` the appropriate way of dealing with this?

Maybe a more general question (asking this because you will use this as the showcase 'right way'), how should we deal with errors on `get` requests?

Reply:
Good question! We don't really handle those at the moment but we can at least take care of the loading state in here. We also don't handle mutation errors very consistently (except in forms) so I'll add a task to i3 to come up with a good mechanism for all of these (perhaps with some help and input from design, as well). 👍